objects/applications/data_hub/aicoe-argocd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: aicoe-argocd
+spec:
+  destination:
+    namespace: aicoe-argocd
+    server: https://api.ocp.prod.psi.redhat.com:6443
+  project: data-hub
```

Comment:
Not sure if ArgoCD is `data-hub` project specific. I admit, I don't know about a better project placement, though...

Reply:
Hrm, that's a good question. I think I just assumed that it would overlap with the data hub team's responsibilities as the infra folks

Reply:
@HumairAK what do you think?

Reply:
yeah I think we should keep it in the datahub project